docs/contributing/getting-started.md

```diff
@@ -0,0 +1,71 @@
+This guide covers how to start contributing to Sedna 😄
+
+## 1. Familiarize Yourself With Contributing to Sedna
+
+### Setup GitHub Account
+
+Sedna is developed on [GitHub][github] and will require
+an account for contribution.
+
+### Check The Kubernetes Contributor Guides
+
+You may come back to this later, but we highly recommend reading these:
+
+- [Kubernetes Contributor Guide](https://git.k8s.io/community/contributors/guide) 
+  - Main contributor documentation, or you can just jump directly to the [contributing section](https://git.k8s.io/community/contributors/guide#contributing)
+- [Contributor Cheat Sheet](https://git.k8s.io/community/contributors/guide/contributor-cheatsheet)
+   - Common resources for existing developers
+
+
+### Read The Sedna Docs 
+
+These [proposals] cover framework and multi edge-cloud synergy capabilities of the project, which may be helpful to review before contributing. 
+
+### Reaching Out
+
+Issues are tracked on GitHub. Please check [the issue tracker][issues] to see
+if there is any existing discussion or work related to your interests.
+
+In particular, if you're just getting started, you may want to look for issues
+labeled <a href="https://github.com/kubeedge/sedna/labels/good%20first%20issue" class="gh-label" style="background: #7057ff; color: white">good first issue</a> or <a href="https://github.com/kubeedge/sedna/labels/help%20wanted" class="gh-label" style="background: #006b75; color: white">help wanted</a> which are standard labels in the Kubernetes
+project.
+The <a href="https://github.com/kubeedge/sedna/labels/help%20wanted" class="gh-label" style="background: #006b75; color: white">help wanted</a> label marks issues we're actively seeking help with while <a href="https://github.com/kubeedge/sedna/labels/good%20first%20issue" class="gh-label" style="background: #7057ff; color: white">good first issue</a> is additionally applied to a subset of issues we think will be particularly good for newcomers.
+See [Kubernetes help wanted] for best practices.
+
+If you're interested in working on any of these, leave a comment to let us know!
+
+If you do not see anything, please [file a new issue][file-an-issue].
+
+> **NOTE**: _Please_ file an enhancement / [feature request issue][file-a-fr] to discuss features before filing a PR (ideally even before writing any code), we have a lot to consider with respect to our
+> existing users and future support when accepting any new feature.
+>
+> To streamline the process, please reach out and discuss the concept and design
+> / approach ASAP so the maintainers and community can get involved early.
+
+Also -- Please reach out in general for bugs, feature requests, and other issues!  
+
+- The issue tracker by [filing an issue][file-an-issue]
```

Comment:
Is "issue tracker" here or "issue is tracked"?

Reply:
I decide to remove it since it's duplicated